OptimoveSDK/optimove-sdk/src/main/java/com/optimove/sdk/optimove_sdk/Optimove.java

```diff
@@ -460,18 +424,104 @@ public void reportScreenVisit(@NonNull String screenName, @Nullable String scree
                 .reportEvent(Collections.singletonList(new SetPageVisitEvent(screenName, screenCategory)));
     }
 
-    public void fcmTokenRefreshed(){
-        optipushManager.syncToken();
+    /**
+     * Clears any existing association between this install record and a user identifier
+     * @see Kumulos#associateUserWithInstall(Context, String)
+     * @see Kumulos#getCurrentUserIdentifier(Context)
+     * @param context
+     */
+    public static void clearUserAssociation(@NonNull Context context) {
+        Kumulos.clearUserAssociation(context);
     }
 
-    public void disablePushCampaigns() {
-        optipushManager.disablePushCampaigns();
+    /**
+     * Returns the identifier for the user currently associated with the Kumulos installation record
+     *
+     * @see Kumulos#associateUserWithInstall(Context, String)
```

Comment:
should check javadocs for references to Kumulos

Reply:
Suggest we compile (assert works) find/replace in files Kumulos -> Optimobile, compile again. At least then we've clobbered all the references and there shouldnt be a namespace / classname conflict.